component/src/main/java/com/iluwatar/component/component/graphiccomponent/ObjectGraphicComponent.java

```diff
@@ -16,12 +16,6 @@ public class ObjectGraphicComponent implements GraphicComponent {
    */
   @Override
   public void update(GameObject gameObject) {
-    if (gameObject.getVelocity() > 0) {
-      LOGGER.info(gameObject.getName() + " has a positive velocity.");
-    } else if (gameObject.getVelocity() < 0) {
-      LOGGER.info(gameObject.getName() + " has a negative velocity.");
-    } else {
-      LOGGER.info(gameObject.getName() + " has zero velocity.");
-    }
+      LOGGER.info(gameObject.getName() + "'s current velocity: "+ gameObject.getVelocity());
```

Comment:
bit more concise and clear with less logs, now it just prints the current velocity - looks good